contracts/bond/DaoBondCollateralWhitelist.sol

```diff
@@ -15,6 +15,9 @@ abstract contract DaoBondCollateralWhitelist is Initializable {
         mapping(address => string) symbols;
     }
 
+    event CollateralAdded(address collateralTokens, string symbol);
+    event CollateralRemoved(address collateralToken, string symbol);
```

Comment:
do we want the symbol here too or superfluous? do we want the address to be `indexed`?

Reply:
@CjHare

Reply:
Yes, we could do indexing of the address. I've not really paid much attention to indexing of the event parameters generally.

Reply:
remove `symbol`

Reply:
`AddCollateral`